analytics/migrations/0001_initial.py

```diff
@@ -87,7 +86,7 @@ class Migration(migrations.Migration):
                 ('value', models.BigIntegerField()),
                 ('anomaly', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='analytics.Anomaly', null=True)),
             ],
-            bases=(zerver.lib.str_utils.ModelReprMixin, models.Model),
+            bases=(models.Model,),
         ),
         migrations.AlterUniqueTogether(
             name='usercount',
```

Comment:
I think it doesn't make sense to change older schema migrations.  We might need a new migration here, though?

http://zulip.readthedocs.io/en/latest/schema-migrations.html is helpful reading

Reply:
Re-added back all the migration schemes and the `ModelReprMixin` class definition. It's just that it is no longer used in the current model